client/src/components/History/Archiving/HistoryArchive.vue

```diff
@@ -50,16 +48,24 @@ async function updateSearchQuery(query: string) {
 
 async function loadArchivedHistories() {
     isLoading.value = true;
-    const result = await fetchArchivedHistories({
-        query: searchText.value,
-        currentPage: currentPage.value,
-        pageSize: perPage.value,
-        sortBy: sortBy.value,
-        sortDesc: sortDesc.value,
-    });
-    totalRows.value = result.totalMatches;
-    archivedHistories.value = result.histories;
-    isLoading.value = false;
+    try {
+        const result = await fetchArchivedHistories({
+            query: searchText.value,
+            currentPage: currentPage.value,
+            pageSize: perPage.value,
+            sortBy: sortBy.value,
+            sortDesc: sortDesc.value,
+        });
+        totalRows.value = result.totalMatches;
+        archivedHistories.value = result.histories;
+    } catch (error) {
+        toast.error(
+            localize(`Failed to load archived histories with reason: ${errorMessageAsString(error)}`),
```

Comment:
Not for this PR, but maybe we should bake that into `errorMessageAsString`

Reply:
You mean the `localize` for all error messages right? :+1:

Reply:
yes

Reply:
I've opened https://github.com/galaxyproject/galaxy/issues/18688 with that